tests/e2e/run-scanner-v4-install.bats

```diff
@@ -253,6 +253,13 @@ setup() {
     _end
 }
 
+# CRD needs to be owned by Helm if upgrading to 4.7+ from 4.6.x via Helm
+apply_crd_ownership() {
```

Comment:
Currently this is only invoked in one case where `stackrox` is the namespace, but please make it take the namespace as a parameter. Also a more specific name would be nice, how about:
```suggestion
apply_config_crd_ownership_for_upgrade() {
```